docs/resources/server/recipes/builtin.md

```diff
@@ -0,0 +1,368 @@
+# Built-In Recipe Types
+
+Minecraft provides a variety of recipe types and serializers out of the box for you to use. This article will explain each recipe type, as well as how to generate them.
+
+## Crafting
+
+Crafting recipes are typically made in crafting tables, crafters, or in modded crafting tables or machines. Their recipe type is `minecraft:crafting`.
+
+### Shaped Crafting
+
+Some of the most important recipes - such as the crafting table, sticks, or most tools - are created through shaped recipes. These recipes are defined by a crafting pattern or shape (hence "shaped") in which the items must be inserted. Let's have a look at what an example looks like:
+
+```json5
+{
+  "type": "minecraft:crafting_shaped",
+  "category": "equipment",
```

Comment:
The category should match the one used in the datagen example and should be one that actually exists, such as `tools`

Reply:
This is directly copied from `iron_pickaxe.json`, the `equipment` category is a thing (apparently).